src/aind_metadata_mapper/stimulus/camstim.py

```diff
@@ -90,44 +86,43 @@ def __init__(
             if self.opto_table_path.exists():
                 self.stim_epochs.append(self.epoch_from_opto_table())
         except Exception:
```

Comment:
What is it that you expect to fail here? Can there be functions to process different types of sessions? I think there are only two and since camstim will be deprecated in the future it should be fine to pull those types out into separate functions so the user knows what is being processed.

Reply:
You're right, this is a bit of messy code and doesn't error out in a clear way. I'll update to it a proper if else since we should only have to worry about ephys or ophys sessions.